.github/workflows/run-workload.yaml

```diff
@@ -28,41 +28,36 @@ jobs:
     runs-on: cfa-cdcgov
     environment: production
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.13"
-
-      - name: cache poetry
-        uses: actions/cache@v4
-        with:
-          path: ~/.local
-          key: ${{ runner.os }}-poetry
-
-      - name: install poetry
-        run: pip install poetry
-
-      - name: install dependencies
-        run: poetry install
-
-      - name: Azure Service Principal CLI login
-        uses: azure/login@v2
-        with:
-          creds: ${{ secrets.EDAV_CFA_PREDICT_NNHT_SP }}
-
-      - name: set pythonpath
-        run: |
-          echo "PYTHONPATH=$PWD" >> $GITHUB_ENV
-
-      - name: Execute workload script
-        run: |
-          state=${{ inputs.state }} \
-          disease=${{ inputs.disease }} \
-          reference_dates=${{ inputs.reference_dates }} \
-          report_date=${{ inputs.report_date }} \
-          data_source=${{ inputs.data_source }} \
-          data_path=${{ inputs.data_path }} \
-          data_container=${{ inputs.data_container }} \
-          production_date=${{ inputs.production_date }} \
-          job_id=${{ inputs.job_id }} \
-            poetry run python pipelines/epinow2/generate_config.py
+    - uses: actions/checkout@v4
+
+    - name: Install uv
+      uses: astral-sh/setup-uv@v5
+      with:
+        enable-cache: true
+
+    - name: "Set up Python"
+      uses: actions/setup-python@v5
+      with:
+        python-version-file: ".python-version"
+
+    - name: Azure Service Principal CLI login
+      uses: azure/login@v2
+      with:
+        creds: ${{ secrets.EDAV_CFA_PREDICT_NNHT_SP }}
+
+    - name: set pythonpath
+      run: |
+        echo "PYTHONPATH=$PWD" >> $GITHUB_ENV
```

Comment:
I'm not sure if this step is still necessary. @amondal2 do you remember why we needed this in the original?

Reply:
We were having issues with PYTHONPATH in the Github runner (the classic ModuleNotFound error), this fixed that.

Reply:
That said, if switching to `uv` removes that error, then we can get rid of this step.

Reply:
We'll give a shot without and see. I'll also try to leave the existing one in place for now.